src/main/java/gregtech/common/covers/CoverConveyor.java

```diff
@@ -473,8 +473,8 @@ public ModularUI createUI(EntityPlayer player) {
                 .setTooltipHoverString("cover.universal.manual_import_export.mode.description"));
         primaryGroup.addWidget(new ToggleButtonWidget(130, 166, 20, 20, GuiTextures.BLOCKS_INPUT, () -> blocksInput, val -> blocksInput = val).setTooltipText("cover.conveyor.blocks_input"));
 
-        TileEntity coverTile = coverHolder.getWorld().getTileEntity(coverHolder.getPos());
-        if (coverTile instanceof TileEntityItemPipe) {
+        if (coverHolder.getWorld().getTileEntity(coverHolder.getPos()) instanceof TileEntityItemPipe ||
+                coverHolder.getWorld().getTileEntity(coverHolder.getPos().offset(attachedSide)) instanceof TileEntityItemPipe) {
```

Comment:
What's the purpose of this change? Seems like it was fine before only checking the world for a TE once

Reply:
Bruh. This fixes the bug where the button shows only up when the cover is on a item pipe. Now it will also show if it is on a block next to a item pipe